code/modules/mob/living/carbon/human/human_helpers.dm

```diff
@@ -122,10 +122,16 @@
 		to_chat(src, span_warning("You can't bring yourself to use a ranged weapon!"))
 		return FALSE
 
+/// Returns true if this mob or one of its arms has chunky fingers
 /mob/living/carbon/human/proc/check_chunky_fingers()
-	if(HAS_TRAIT_NOT_FROM(src, TRAIT_CHUNKYFINGERS, RIGHT_ARM_TRAIT) && HAS_TRAIT_NOT_FROM(src, TRAIT_CHUNKYFINGERS, LEFT_ARM_TRAIT))
+	if (HAS_TRAIT(src, TRAIT_CHUNKYFINGERS))
 		return TRUE
-	return (active_hand_index % 2) ? HAS_TRAIT_FROM(src, TRAIT_CHUNKYFINGERS, LEFT_ARM_TRAIT) : HAS_TRAIT_FROM(src, TRAIT_CHUNKYFINGERS, RIGHT_ARM_TRAIT)
+
+	var/obj/item/bodypart/arm/using_arm = hand_bodyparts[active_hand_index]
```

Comment:
You can just use `get_active_hand` for this, which in turn means there's no reason for this to be human level, because all the hand API is on `/mob` level

Reply:
`get_active_hand` doesn't respect the idea that you might have more than two hands and also "don't put a proc on the living level" started this entire conversation to begin with (although tbh i disagreed that we shouldn't do that so if after looking at this more it makes sense to put it there I will)

Reply:
If we make it a generic proc then we don't even need a snowflake mob level proc, we can just call it directly from `/can_melee_attack_chain_with`